### ISSUE TITLE
Incompatibility with my plugin

### ISSUE THREAD

**Issue by marlester-dev** [NEZNAMY/TAB#910]
### Server version

Spigot 1.19.4

### TAB version

3.3.2

### Describe the bug

TAB plugin does not work properly with [my Really Fake Players](https://github.com/marlester-dev/ReallyFakePlayers) plugin, it's likely that TAB does not properly recognise fake players and tab ordering, ping, etc. does not work.

### Expected behavior

TAB recognises fake players and tab ordering, ping, etc. work.

### Steps to reproduce

1. Install both plugins
2. Add fake players
3. 

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [ ] I ticked all of the boxes without actually reading them
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by NEZNAMY** [comment]
How is the plugin supposed to recognize completely random packets sent by another plugin?

**Comment by Tanguygab** [comment]
if that's the case, then try to add TAB to your plugin.yml's loadbefore so TAB loads after your plugin.

**Comment by Tanguygab** [comment]
try creating a new [BukkitTabPlayer](https://github.com/NEZNAMY/TAB/blob/master/bukkit/src/main/java/me/neznamy/tab/platforms/bukkit/BukkitTabPlayer.java#L66) for each fake player and adding them to TAB with [TAB#addPlayer(TabPlayer)](https://github.com/NEZNAMY/TAB/blob/master/shared/src/main/java/me/neznamy/tab/shared/TAB.java#L207) (you need to import the jar though because those aren't in the API)

**Comment by Tanguygab** [comment]
also send your server logs, just to see if your plugins did load before TAB or not (and your plugin.yml as well to make sure it's correct)

**Comment by NEZNAMY** [comment]
Call PlayerJoinEvent and TAB will handle it.
